specification/implementation/metadata/mets/filesec/index.md

```diff
@@ -1,8 +1,10 @@
 ### 5.3.5.	Use of the METS file section (element fileSec)
-Use of the METS fileSec element is highly recommended by the CSIP (although not mandatory). It should describe all components of the IP which have not been already included in the amdSec and dmdSec elements. For all files the location and checksum need to be available. The METS file section serves as a manifest, allowing users to check that all files are present and a package is complete as well as testing the integrity of package files using checksum values.
+Use of the METS fileSec element is highly recommended by the CSIP (although not mandatory). It should describe all components of the IP which have not been already included in the amdSec and dmdSec elements. For all files the location and checksum need to be available.
```

Comment:
The CSIP does not make the use of the METS fileSec element mandatory, but it is strongly recommended. The fileSec should describe every component of the IP not already described in the amdSec and dmdSec elements. Location and checksum values must be provided for all file entries.